.github/workflows/pr-title.yml

```diff
@@ -0,0 +1,21 @@
+name: PR title checks
+
+on:
+  pull_request:
+    types: [opened, edited, reopened]
+
+jobs:
+  pr-title:
+    name: Lint PR Title
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: taiki-e/install-action@v2
+        with:
+          tool: cargo-binstall
+      - run: cargo +stable binstall committed
```

Comment:
using +stable here, otherwise this first downloads the nightly toolchain specified in our `rust-toolchain.toml`.